lib/utils.pm

```diff
@@ -107,6 +108,10 @@ sub wait_boot {
             die "needle 'grub2' not found";
         }
     }
+    else {
+        # With Xen PV we don't see the graphical bootloader
+        record_soft_failure 'missing implementation';
```

Comment:
Well, in my case openqa won't read from serial nor VNC after restart on Xen PV (aka Xen Linux). I doubt it's a product problem or test problem. Most probably some os-autoinst limitation as it won't - perhaps - reconnect to serial/VNC. I'll remove the `else { }` part then.